.drone.star

```diff
@@ -271,10 +368,45 @@ def consumerTestPipeline(subFolderPath = '/'):
             pactConsumerTests(True if subFolderPath == '/' else False),
     }
 
-def providerTestPipeline():
+def ocisProviderTestPipeline():
     return {
         'kind': 'pipeline',
-        'name': 'testProvider',
+        'name': 'testOCISProvider',
+        'platform': {
+            'os': 'linux',
+            'arch': 'amd64'
+        },
+        'workspace': {
+            'base': '/var/www/owncloud',
+            'path': 'owncloud-sdk'
+        },
+        'trigger': {
+            'branch': 'master'
+        },
+        'depends_on': [ 'testConsumer-root', 'testConsumer-subfolder' ],
+        'steps':
+            buildSystem() +
+            pactLog() +
```

Comment:
This step is useless since we moved to pactv3 but can be removed in different PR

Reply:
@dpakach PR in https://github.com/owncloud/owncloud-sdk/pull/741